### ISSUE TITLE
[Question] there a way to lock a Deity, how would one unlock it? (coding related)

### ISSUE THREAD

**Issue by Child-0f-Night** [skyjay1/RPG-Gods#20]
I found this mod not long ago, and liked the idea of it as well as it's potential and versatility (at least at face value). I haven't played the mod yet because I want to make a data pack before playing it (my reasoning for this was the amount of deities / pantheons to choose from, or lack there of). so i go to make the data pack, and apparently there is a mechanic to unlock deities at some point after the game starts, in the form of the line of code (everything within the following Parentheses): ("unlocked": [true/false],). 

according to the wiki, the true/false statement means whether the the deity starts off as unlocked from the moment the world begins, or if they are unlocked by other means(?) the wiki wasn't vary clear on that, and i glossed through the rest of the wiki, and couldn't find anything other then the mechanic can be used with Alters too. 

______________________________________________________________________________________________________________________________________________
in summary, apparently there is an option to lock a deity from the start of the game. how do i code the data pack to give a way to "unlock" a god manually? or was there already a way that doesn't need it in the data pack and I'm just in the dark about it?
______________________________________________________________________________________________________________________________________________

(also side note: this is my first ever data pack, fortunately the prosses is easy to learn for me with the power of you tube tutorials, this was really the only road block i had. but still do consider my lack of data pack experience, i can give more detail on my proficiency level with code if needed.)

**Comment by skyjay1** [comment]
There are 2 ways to unlock a deity who started out “locked”, using a data pack:
1. A perk for some other (unlocked) deity has the “unlock” [perk action](https://github.com/skyjay1/RPG-Gods/wiki/Perks#perk-action) type, triggered by some [perk condition](https://github.com/skyjay1/RPG-Gods/wiki/Perks#perk-condition)
2. An mcfunction file runs the [favor command](https://github.com/skyjay1/RPG-Gods/wiki/Favor#commands) `/favor set <targets> <deity> unlocked (true|false)`

**Comment by Child-0f-Night** [comment]
thank you!
